src/mainwindow.cpp

```diff
@@ -520,6 +519,7 @@ MainWindow::MainWindow(Settings &settings
 
   QApplication::instance()->installEventFilter(this);
 
+  scheduleCheckForProblems();
```

Comment:
Maybe schedule it a bit earlier since it will be a background task in the end? I don't know if it would change anything in practice though...

Reply:
I put it towards the bottom in case any of the other things that were getting set up were important for the diagnostic plugins.

Reply:
Hm, ok. I really think there should be a better distinction between core stuff (`OrganizerCore`) and UI stuff (`MainWindow`), but that's out-of-scope for this PR.